src/analyzer.ts

```diff
@@ -18,16 +20,14 @@ export class Analyzer extends AnalyzerGen {
 
 		this.analyzerProcess.stdout.on("data", (data: Buffer) => {
 			let message = data.toString();
-			console.log(`RCV: ${message}`);
+			if (verbose && message.trim().length != 0) console.log(`<== ${message}`);
```

Comment:
I moved the if bodies to their own lines. Happy to adjust to whatever coding style - definitely more important that the code in a repo shares a similar style.